example/radar/radar_scan_precip.py

```diff
@@ -37,27 +39,20 @@
 log = logging.getLogger()
 
 
-def plot(data: wrl.io.XRadVolume):
+def plot(data: xr.Dataset):
     """Plot radar data with prefixed settings."""
     # Get first sweep in volume.
-    swp0 = data[0].data
+    swp0 = data.isel({"time": 0})
 
     # Georeference Data.
-    swp0 = swp0.pipe(wrl.georef.georeference_dataset)
+    swp0 = swp0.pipe(xradar.georeference.add_crs)
 
     # Plot and display data using cartopy.
     fig = plt.figure(figsize=(20, 8))
     ax1 = fig.add_subplot(121, aspect="equal")
-    swp0.DBZH[0].plot(x="x", y="y", ax=ax1)
+    swp0.DBZH.plot(x="x", y="y", ax=ax1)
```

Comment:
Dear @kmuehlbauer , after changing to xradar variables DBZH and VRADH seem to not be recognized automatically anymore. Did I do something wrong?